### PR TITLE
ldap: add broader introduction to the protocol

### PR DESCRIPTION
**Pull request by pbrezina**

### CODE REVIEW

File: src/docs/ldap/ldap-introduction.rst [hunk 1]
```diff
@@ -1,31 +1,206 @@
...
+    optional attributes can be set on the entry. Object classes are defined in
+    schemas that are installed on the LDAP server.
+
+Attributes
+    Attributes are single or multi-valued properties of the entry. Their format
+    and functions (e.g. date, string, case sensitiveness, comparison schema,
+    limits, etc.) are defined by schema and object classes. Some standard and
+    common attributes are:
+
+    * **dc**: domain component -- used to identify the directory domain *
```
Comment (justin-stephenson) [inline src/docs/ldap/ldap-introduction.rst:77]: intended trailing `*` here?

File: src/docs/ldap/ldap-introduction.rst [hunk 2]
```diff
@@ -1,31 +1,206 @@
...
+    * **ou**: orgnanizational unit -- used to split entries into named containers
```
Comment (justin-stephenson) [inline src/docs/ldap/ldap-introduction.rst:78]: typo: organizational

**Review by justin-stephenson — CHANGES_REQUESTED** [review]
Minor changes requested.

File: src/docs/ldap/ldap-introduction.rst (hunk 1 above)
Comment (pbrezina) [reply src/docs/ldap/ldap-introduction.rst:77]: Nope.

**Comment by pbrezina** [comment]
Thank you. See updated patch.

File: src/docs/ldap/ldap-introduction.rst [hunk 3]
```diff
@@ -1,31 +1,206 @@
...
+    To read more about how SSSD is used in LDAP integration at a high level, refer to the following links:
+
+        * `Understanding SSSD and its benefits <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_authentication_and_authorization_in_rhel/understanding-sssd-and-its-benefits_configuring-authentication-and-authorization-in-rhel>`_
+        * `Configuring SSSD to use LDAP and require TLS authentication <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_authentication_and_authorization_in_rhel/configuring-sssd-to-use-ldap-and-require-tls-authentication_configuring-authentication-and-authorization-in-rhel>`_
+
+LDAP Basics
+***********
+
+The rest of the page is dedicated to readers that are not yet familiar with the
+directory servers. It explains the very basics that will help you dive into the
```
Comment (justin-stephenson) [inline src/docs/ldap/ldap-introduction.rst:48]: s/the directory servers/directory servers/

**Comment by justin-stephenson** [comment]
Ack with the last single change.

**Comment by pbrezina** [comment]
Done. Thank you.

**Pull request merged**